src/actions/types.js

```diff
@@ -1,17 +1,22 @@
-// Registration action types
 export const REGISTER_USER_SUCCESS = 'REGISTER_USER_SUCCESS';
 export const REGISTER_USER_ERROR = 'REGISTER_USER_ERROR';
 export const LOGIN_USER_SUCCESS = 'LOGIN_USER_SUCCESS';
 export const LOGIN_USER_ERROR = 'LOGIN_USER_ERROR';
 export const SOCIAL_LOGIN_INITIATED = 'SOCIAL_LOGIN_INITIATED';
 export const SOCIAL_LOGIN_SUCCESS = 'SOCIAL_LOGIN_SUCCESS';
-export const GET_ALL_ARTICLES_SUCCESS = 'GET_ALL_ARTICLES_SUCCESS';
 
-// Profile action types
 export const GET_PROFILE_PAYLOAD = 'GET_PROFILE_PAYLOAD';
 export const GET_PROFILE_ERROR = 'GET_PROFILE_ERROR';
 export const GET_PROFILE_INITIATED = 'GET_PROFILE_INITIATED';
+
 export const SEND_RESET_LINK_SUCCESS = 'SEND_RESET_LINK_SUCCESS';
 export const SEND_RESET_LINK_ERROR = 'SEND_RESET_LINK_ERROR';
 export const RESET_PASSWORD_SUCCESS = 'RESET_PASSWORD_SUCCESS';
 export const RESET_PASSWORD_ERROR = 'RESET_PASSWORD_ERROR';
+
+export const GET_ALL_ARTICLES_INITIATED = 'GET_ALL_ARTICLES_INITIATED';
```

Comment:
You can remove lines 18 and 19 because I did not use them for fetching articles.

Reply:
@mendozabree Lines 18 and 19 have been removed